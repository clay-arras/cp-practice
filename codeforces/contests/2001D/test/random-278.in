1
20
11 3 13 8 21 4 17 5 12 7 11 10 20 4 3 19 18 1 11 15
