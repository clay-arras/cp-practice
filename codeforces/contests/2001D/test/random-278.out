16
13 8 21 4 17 5 12 7 11 10 20 3 19 18 1 15
